website/docs/03_components/03_package-repository.md

```diff
@@ -24,7 +22,32 @@ A package repository must use the following directory structure to be fully comp
 
 The root `index.yaml` contains a list of all packages available from this repository. It is used primarily by client software to aid explorability.
 All files related to a package reside in a directory that must have the same name as the package. 
-Inside a package's directory there may be a `versions.yaml` that contains a list of all versions available for this package.
-If such a `versions.yaml` file exists, there must be a subdirectory for each version containing a `package.yaml` file.
-If no `versions.yaml` file exists, there must be a `package.yaml` in the package's directory.
+Inside a package's directory there must be a `versions.yaml` that contains a list of all versions available for this package.
+There must be a subdirectory for each version containing a `package.yaml` file.
 A `package.yaml` contains a manifest of that package which holds information such as longer descriptions and included files.
+
+### Version Numbers
+
+The version number of a package must follow the [semver specification](https://semver.org), with the additional constraint that the build number of a version is only allowed to consist of digits. 
+Although the specification states that [build numbers must be ignored](https://semver.org/#spec-item-10) when determining version precedence, 
+we see no other way than to do so. Therefore, we only allow digits in the build number, such that we can decide which version is newer. 
+
+This is important for Glasskube to distinguish between two different versions of a package manifest, that might have the same underlying software version of a package.
+For example, there could be a package  `kubernetes-dashboard` in version `2.7.0`. 
+The preferred way to make this version available in the Glasskube package repository, is to create a version `2.7.0+1`. 
+If the package manifest turns out to be faulty and need to be corrected (e.g. typos, wrong entrypoints), the maintainer needs to add a version `2.7.0+2`. 
```

Comment:
I would limit the increase of the build number to "errors". We might also increase it for metadata updates, dependency changes or any other update to the package it self that does not change the app versions